pkg/metrics/metrics.go

```diff
@@ -308,6 +310,15 @@ var (
 		Help: "Duration in seconds of the garbage collector process " +
 			"labeled by datapath family and completion status",
 	}, []string{LabelDatapathFamily, LabelStatus})
+
+	// Errors and warnings
+
+	// ErrorsWarnings is the number of errors and warnings in cilium-agent instances
+	ErrorsWarnings = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
The metric name is misleading as it will count for the number of `errors`, `warnings`, `panics` and `fatals` .

Reply:
Yes, but `ErrorsWarningsPanicsFatals` seems bit weird for me and I thought that we may consider panics and fatals as specific kind of errors.

Reply:
Would we ever see the increment for a `panic` or `fatal` ? I think both kill the program inside logrus, and it's unlikely that prometheus would scrape at that instant.

Reply:
Good point. I will just remove `panic` and `fatal`.

Reply:
done